pkg/client/transport.go

```diff
@@ -68,6 +68,10 @@ func NewBearerAuthRoundTripper(bearer string, rt http.RoundTripper) http.RoundTr
 }
 
 func (rt *bearerAuthRoundTripper) RoundTrip(req *http.Request) (*http.Response, error) {
+	if len(req.Header.Get("Authorization")) != 0 {
```

Comment:
At the very least, can you repeat your commit message as a comments here, and better yet, add it as a godoc for NewBearerAuthRoundTripper?